src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -125,7 +125,11 @@ void SX127xDriver::SetBandwidthCodingRate(SX127x_Bandwidth bw, SX127x_CodingRate
     if (currSF == SX127x_SF_6) // set SF6 optimizations
     {
       hal.writeRegister(SX127X_REG_MODEM_CONFIG_1, bw | cr | SX1278_HEADER_IMPL_MODE, SX12XX_Radio_All);
-      hal.writeRegisterBits(SX127X_REG_MODEM_CONFIG_2, SX1278_RX_CRC_MODE_OFF, SX1278_RX_CRC_MODE_MASK, SX12XX_Radio_All);
+      #if defined(RADIO_SX1272)
+        hal.writeRegisterBits(SX127X_REG_MODEM_CONFIG_1, SX1278_RX_CRC_MODE_OFF, SX1278_RX_CRC_MODE_MASK, SX12XX_Radio_All);
```

Comment:
Should this be `SX1272_RX_CRC_MODE_OFF, SX1272_RX_CRC_MODE_MASK` ?

If you are feeling enthusiastic (not required for me to approve this PR) `SetBandwidthCodingRate` could really do with a further clean up.  Pull the CRC config into its own function, and use SX127x_SF_6 to just force headerExplMode false.

Reply:
Yeah this is pretty confusing. I just fixed it, and I'll take a look at your suggestion

Reply:
I took your advice and broke out CRC config into its own function to simplify `SetBandwidthCodingRate`